test/tall/statement/for_comment.stmt

```diff
@@ -69,8 +69,7 @@ for  (  init
 <<<
 ### Looks weird, but users don't tend to put comments here.
 for (
-  init
-  // comment
+  init// comment
```

Comment:
Do we want a little bit of spacing (1 space) when we move a comment like this?

Reply:
Ideally, it would be a newline as in the previous code. I tried a bunch last night and couldn't figure out a clean way to get that working. But I gave it another shot today and found a solution, so done. :)

Thanks for pushing on this.